src/uwtools/drivers/jedi.py

```diff
@@ -8,6 +8,7 @@
 from iotaa import asset, refs, run, task, tasks
 
 from uwtools.drivers.jedi_base import JEDIBase
```

Comment:
Question on this, I saw the other file was also modified for `src/uwtools/drivers/jedi_base.py`. Here we're importing the JEDIBase class and then setting the `set_driver_docstring(JEDI)` at the end of this file, but we don't do the same action for `jedi_base.py`. My thought process was the jedi driver here inherited from the jedi base class. 

Would it make sense to move the `set_driver_docstring(JEDI)` to the `JEDIBase` Class instead something like `set_driver_docstring(JEDIBase)`and have the `JEDI` Class inherit from it? If you have an explanation based on the choice that'd be great!

Reply:
Good question. The `IODA` class has a docstring that provides the header line
```
A driver for the IODA component.
```
and the JEDI driver provides header line
```
A driver for the JEDI component.
```
The new `set_driver_docstring()` function, called at the bottom of each (non-base -- so not `jedi_base.py` or `mpas_base.py`) driver module assembles the driver's docstring by combining 1. The header line from the drive, like those shown above, with the parameter descriptions from the parent class. So I think it's necessary to do this on the final driver class, and not on the base class, as otherwise the appropriate header line will not be available.

Reply:
Great question! I appreciated the answer, too. :)